src/local-editor-service.ts

```diff
@@ -98,21 +97,45 @@ export class LocalEditorService extends EditorService {
           };
         })
       };
-    } else if (location.kind === 'attribute') {
+    }
+
+    if (location.kind === 'attributeValue') {
+      const domModules = document.getByKind('dom-module');
+      if (domModules.size > 1) return;
+      const domModule = domModules.values().next().value;
+      if (!domModule || !domModule.id) return;
+      const outerElement = document.getOnlyAtId('element', domModule.id);
+      if (!outerElement) return;
+      const sortPrefixes = this._createSortPrefixes(outerElement);
+      const innerElement = document.getOnlyAtId('element', location.element.nodeName)!;
+      const innerAttribute = innerElement.attributes.find((value) => value.name === location.attribute)!;
+      return {
+        kind: 'attribute-values',
+        attributes: outerElement.attributes.map(p => {
+          const sortKey =
+              (sortPrefixes.get(p.inheritedFrom) || `ddd-`) + p.name;
+          let autocompletion: string;
+          if (innerAttribute.notify) {
```

Comment:
it's possible that the attribute value already has expression delimiters. Once we have a solid expression parser we can do much more, but for now let's check whether the attribute value matches `/^\s*(\{\{/|\[\[)/` and only add expression delimiters if they're not found